argo-cd-apps/base/member/infra-deployments/multi-platform-controller/multi-platform-controller.yaml

```diff
@@ -15,8 +15,8 @@ spec:
                 clusterDir: ""
           - list:
               elements:
-                - nameNormalized: stone-prod-m01
-                  values.clusterDir: stone-prod-m01
+                - nameNormalized: stone-prd-m01
+                  values.clusterDir: stone-prd-m01
```

Comment:
you also need to rename the folder stone-prod-m01 to stone-prd-m01 in the component folder